docs/releasing/testing-and-linting.md

```diff
@@ -47,3 +47,13 @@ If a snapshot test fails, review the difference in the console. If the change is
 `npm test -- -u src/govuk/components/button`
 
 This will update the snapshot file. Commit this file separately with a commit message that explains you're updating the snapshot file and an explanation of what caused the change.
+
+## Visual regression testing with Percy
+
+We use [Percy](https://percy.io/), a visual regression testing tool, to generate and store screenshots of our components to help us check for any unintended visual changes.
+
+We generate two screenshots for each default example of every component: one with JavaScript enabled and one with JavaScript disabled. Screenshots are not taken for all the different variations of each component. This tool is not a replacement for manual testing.
+
+The screenshots are public, so they can be checked without logging in. A BrowserStack account is needed to approve or reject any changes. It's the responsibility of the person reviewing the pull request code to approve any visual changes that Percy highlights.
```

Comment:
I can access these when logged in, but am struggling to figure out how in an incognito window.

Created a dummy account for Github so I could view the Github Action logs and head off to the Percy build. Created a Browserstack account, but am getting "Access Denied" on the build at https://percy.io/55ba5e1a/govuk-frontend/builds/15872008

![Screenshot 2022-02-23 at 09 57 56](https://user-images.githubusercontent.com/22524634/155297056-137937d3-3b61-45ba-9b94-f6b825394022.png)

Reply:
@domoscargin Doh, hadn't enabled that setting. Do you want to give it a try now?

Reply:
That works! 🎉  No login necessary.